guides/source/active_record_querying.md

```diff
@@ -199,6 +199,8 @@ SELECT * FROM customers WHERE (customers.id IN (1,10))
 
 WARNING: The `find` method will raise an `ActiveRecord::RecordNotFound` exception unless a matching record is found for **all** of the supplied primary keys.
 
+If your table uses a composite primary key, you'll need to pass an array to find a single item. You can also pass an array of arrays to find multiple records.
```

Comment:
I think this should include an example. Can you open a new PR that shows the find and sql generated like the above examples?

Reply:
That's a great idea. Will do.